lib/show.rb

```diff
@@ -0,0 +1,17 @@
+class Show
+  attr_reader :name, :creator, :characters
+
+  def initialize(name, creator, characters)
+    @name = name
+    @creator = creator
+    @characters = characters
+    @total_salary = 0
+  end
+
+  def total_salary
+    @characters.sum do |character|
+      character.salary
+    end
```

Comment:
great use of .sum here. Remember that you can also replace the `do` and `end` with curly braces to do this on a single line: `@characters.sum {|character| character.salary}`